domains/x93.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "xavzen",
+           "email": "",
+           "discord": "1124747549450977280",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.imphI7TJ5piJal5QDZrae4MZOp5Qng5f9WDAiEKpI142edS-w_kZjSPLi4JGqJpOIB0HL7EZMKb9uzkkVSL263GrEARaCmlCz3be0Llw9CoJGokIoUlhuBMlZeIE0XWWMsVeYAmU3oxIEPblc9JPs11oeTv1xh2x_b2uEZIry3wP3evd9HPRWivyOHjsyoUeEJx9_okuoJ4ULSS0ZPg0D_wq6tVGA-HwMCg-wUmeqk5DWt_GIJx14hGWGzJrVF9e-b0tPt7iqTN_PH74856kmTuu6_P3K1fspLNqiujiiByYkP8H0sRW9gciBhUGZr925h9IdsHKvXyfd9HtDNgGeg.1DfQ7ov9yg7ggUGRZJqhmw.0tTEyez485OFHSzTbhOsySXdCiE6S0IAzOVi69pbmA_qZco42OYArEf3MHhNQM6p3Zl5Dz-NztfMB-z4tY1xqUCaOme8TgTpb-gYWXc2oh4.K1RLSfDaW92wTf2Lz2cnPQ"
+        },
+    
+        "record": {
+            "CNAME": "https://guns.lol/tylergc"
```

If you want to use CNAME it should be like this
```suggestion
            "CNAME": "guns.lol"
```